airbyte-integrations/connectors/source-xero/source_xero/manifest.yaml

```diff
@@ -0,0 +1,2826 @@
+version: 0.78.5
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: CustomRecordExtractor
+      class_name: source_xero.components.CustomExtractor
+      field_path: ["{{ parameters.extractor_path }}"]
+
+  requester:
+    type: HttpRequester
+    url_base: "https://api.xero.com/api.xro/2.0/"
+    http_method: "GET"
+    authenticator:
+      type: "BearerAuthenticator"
+      api_token: "{{ config['access_token'] }}"
+    request_headers:
+      "Xero-Tenant-Id": "{{ config['tenant_id'] }}"
+      "Accept": "application/json"
+    error_handler:
+      type: CompositeErrorHandler
+      error_handlers:
+        - type: DefaultErrorHandler
+          response_filters:
+            - type: HttpResponseFilter
+              action: IGNORE
+              predicate: >-
+                {{ 'App Only access is not allowed' in response
+                }}
+              http_codes:
+                - 403
+              error_message: Resource is forbidden, Please check all the scopes in xero platform for getting access of resource
+
+  paginator_base:
+    type: "DefaultPaginator"
+    page_size_option:
+      type: "RequestOption"
+      inject_into: "request_parameter"
+      field_name: "page_size"
+    pagination_strategy:
+      type: "PageIncrement"
+      page_size: 5
+    page_token_option:
+      type: "RequestOption"
+      inject_into: "request_parameter"
+      field_name: "page"
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "#/definitions/requester"
+
+  incremental_sync_base:
```

Comment:
Why we don't use `If-Modified-Since` header logic for incremental sync as it was before and filter data instead? It can be a big regression of sync time for users with a lot of data.

Reply:
There is a fault in the API precision as it only supports date precision right now, we need a second precision for incremental reads. Mentioned in  https://github.com/airbytehq/airbyte/pull/36956/files#diff-ad6ad0ce85c924cf3ece039ff15425fd7c28cd5df20d06343b6fe8a5bd67b799R6